discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -199,14 +204,25 @@ def wait_for_install(
         )
 
     def prepare_for_install(
-        self, 
-        controller,
-        iso_download_path=env_variables['iso_download_path'],
-        ssh_key=env_variables['ssh_public_key'],
-        nodes_count=env_variables['num_nodes'],
-        vip_dhcp_allocation=env_variables['vip_dhcp_allocation'],
-        cluster_machine_cidr=env_variables['machine_cidr']
-        ):
+            self,
+            controller,
+            iso_download_path=None,
+            ssh_key=None,
+            nodes_count=None,
+            vip_dhcp_allocation=None,
+            cluster_machine_cidr=None):
+
+        if iso_download_path is None:
+            iso_download_path = self._env["iso_download_path"]
```

Comment:
if it is lets make it :
iso_download_path = iso_download_path or self._env["iso_download_path"]

Reply:
will look prettier